### ISSUE TITLE
Caching behavior

### ISSUE THREAD

**Issue by ktk** [lindas-uc/lindas-issues#8]
I find the caching behavior in the web interfaces in general unpredictable. If I upload new data to the graph it is very hard to figure out if this worked or not. More often than not I get old data or no data. After countless shift-reloads and/or private windows it usually works.

This is IMHO not proper behavior unless caching headers demand this. If they do I would like to know more about the default caching behavior.

**Comment by martin-voigt** [comment]
I know that there are some less predictable problems regarding caching of AngularJS with Firefox. Can you describe 1) which browser you are using and 2) on which pages there are the problems.
edit: in your email you say "on the endpoint", so you mean /sparql ?

**Comment by ktk** [comment]
Yes I mean /sparql. I usually work in Firefox or Chrome directly on /sparql.

**Comment by jonasch** [comment]
Caching behaviour is result of using the Virtuoso Web Interface. We should check, if there is a Configuration for Virtuoso disabling this behaviour. Otherwise http://lindas-data.ch/#/developer/sparql should be usable without any caching issues.
